frontend/src/exams/generate-financial-report-modal.vue

```diff
@@ -16,47 +16,99 @@
       </b-row>
       <b-row class="my-1">
         <b-col sm="4"><label>Start Date:</label></b-col>
-        <b-col sm="6"><b-form-input id="startDate" type="date" v-model=startDate></b-form-input></b-col>
+        <b-col sm="6">
+          <b-form-input id="startDate"
+                        type="date"
+                        v-model=startDate />
+          <b-form-invalid-feedback></b-form-invalid-feedback>
```

Comment:
Does this `form-invalid-feedback` tag show errors with both start and end dates? If so, should it be at the top of the form above all inputs?

Reply:
nope this was an attempt at form validation before we talked on tuesday, and forgot to remove them. thanks for catching that, they've been removed.